crates/apollo-compiler/test_data/diagnostics/0054_variables_are_input_types.graphql

```diff
@@ -0,0 +1,86 @@
+query takesCat($cat: Cat) {
```

Comment:
https://github.com/apollographql/apollo-rs/pull/452 added `0054_*` test data so this PR should use 0055 and 0056

(maybe we can add a check to the test runner that each number is unique)

Reply:
we probably should 🤔 becomes annoying to track manually